templates/gitlab-multi-runner/config.yml

```diff
@@ -0,0 +1,5 @@
+name: gitlab-multi-runner
+description: |
+  a Gitlab pipelines multi-runner, that will spawn privates runners in your infra.
+version: 1.0.0
```

Comment:
Probably, version could be same as gitlab docker image version... 9.3.0 or used version...